checker/tests/index/Issue5471.java

```diff
@@ -0,0 +1,24 @@
+// Test case for https://github.com/typetools/checker-framework/issues/5471.
+
+import org.checkerframework.checker.index.qual.IndexFor;
+
+public class Issue5471 {
+  private static boolean atTheBeginning(@IndexFor("#2") int index, String line) {
+    return (index==0);
+  }
+
+  private static boolean hasDoubleQuestionMarkAtTheBeginning(String line) {
+    int i = line.indexOf("??");
+    if (i != -1) {
+      // Unexpected warning on this line: i is definitely an index for line.
```

Comment:
This comment refers to before the fix.  I would clarify that, or (more likely) just remove the comment.